modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -79,6 +79,16 @@ You can create your Anypoint VPC inside any of the available CloudHub regions. +
 The recommended region to use might vary depending on how you connect to your Anypoint VPC. If you are using a VPN tunnel, you might want to choose the CloudHub region closest to your data center. +
 However, if you are peering with your private AWS VPC, you need to create your Anypoint VPC in the same AWS region.
 
+== FAQ: How to deploy or move applications to a VPC
+
+MuleSoft applications are deployed to an environment (dev, sandbox, and so on). 
+
+When the deployment region matches the VPC region:
```

Comment:
Is "deployment region" the same as "environment"? If so, consider choosing the more precise of the two terms, in this context, and using it only, to avoid confusion.

Reply:
No, they are two separate things.